### ISSUE TITLE
option or envvar to skip LANG='C' check?

### ISSUE THREAD

**Issue by heavywatal** [r-lib/testthat#1563]
I have `LANG='C'` in my .Renviron to disable locales, and `testthat::expect_true()` suddenly started blaming it. How can I solve this warning?

```
Warning (test-write.R:5:3): save_as() works
Changing language has no effect when envvar LANG='C'
Backtrace:
 1. testthat::expect_true(file.exists(path))
      at test-write.R:5:2
 2. testthat:::expect_waldo_constant(act, TRUE, info = info)
 3. testthat:::waldo_compare(...)
 4. reporter$local_user_output()
 5. testthat::local_reproducible_output(...)
 6. withr::local_language(lang, .local_envir = .env)
```

**Comment by heavywatal** [comment]
Closing this because the issue seems to have been solved (in testthat, withr, waldo, or else?)

**Comment by heavywatal** [comment]
It has not been solved yet.

**Comment by heavywatal** [comment]
#1566 may be a duplicate

**Comment by hadley** [comment]
Duplicate of #1566

**Comment by heavywatal** [comment]
#1662 solved #1566, but current `main` still emits warnings with envvar `LANG='C'`.
